repo/packages/D/dcos-ui/0/resource.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images": {
+    "icon-small": "icon-small-placeholder.png",
```

Comment:
@leemunroe do we have any timeline for the icons?

Reply:
Default is fine since this is something users won't see